test/MUXControlsTestApp/MSTest/MUXControlsTestApp.csproj

```diff
@@ -7,7 +7,7 @@
     <GenerateAppInstallerFile>False</GenerateAppInstallerFile>
     <AppxAutoIncrementPackageRevision>False</AppxAutoIncrementPackageRevision>
     <AppxBundle>Never</AppxBundle>
-    <AppInstallerUpdateFrequency>1</AppInstallerUpdateFrequency>
+    <AppInstallerUpdateFrequency>0</AppInstallerUpdateFrequency>
```

Comment:
Where did those changes come from?

Reply:
I don't know what this is... I'll revert it. maybe @llongley does.

Reply:
I believe these are set when you make an AppX from the "publish" option in VS.  I don't know what they do though.